instrumentation/aws-lambda-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambda/v1_0/LambdaUtils.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awslambda.v1_0;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import java.util.concurrent.TimeUnit;
+
+final class LambdaUtils {
+
+  static void forceFlush() {
+    OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
+    if (openTelemetry instanceof OpenTelemetrySdk) {
+      ((OpenTelemetrySdk) openTelemetry)
+          .getSdkTracerProvider()
+          .forceFlush()
```

Comment:
oh no, you found a way to get around the obfuscation!

Reply:
Hmm - yeah I guess we only obfuscate `getTracerProvider`, but not the SDK itself. Is that a bug? :)

Anyways, these classes need to be refactored to accept an SDK as a parameter in some way rather than only using the global, will try to do that soon.